configurations/performance/drivers/java-driver/cassandra-stress-cmds-predefined-steps-for-java-driver.yaml

```diff
@@ -0,0 +1,20 @@
+stress_cmd_r: [
```

Comment:
Why do we need them to be defined again?
Only for the shorter duration?
Can't we provide the duration from outside (e.g. generic duration?)

Reply:
@roydahan @fruch 
There is `stress_duration` parameter. But this parameter set time of execution for stress commands from stress_cmd parameters and is used in test duration calculation. In this test the duration is of step, not test duration.
To be able to provide step duration I need to add new parameter. 
Commonly it is not the problem, I can add such parameter

Reply:
done